src/ORM/ResettableEntityManager.php

```diff
@@ -59,7 +59,7 @@ public function getRepository($className): ObjectRepository
     /**
      * {@inheritDoc}
      */
-    public function createQuery($dql = '')
+    public function createQuery($dql = ''): Query
```

Comment:
Why? I don't think so. There aren't any return types in doctrine/orm right now. There is just notice that it will be added in the future. PS: doctrine/orm is locked to ^2.10 and newest version is 2.11.2.